src/app/routes/objective/sagas/index.js

```diff
@@ -34,6 +37,19 @@ function* fetchDetailSaga({payload: {key}}) {
     }
 }
 
+function* innerFetchItemLeaderboardSaga(jwt, {payload: {key}}) {
+    const {res, error, status} = yield call(fetchItemLeaderboardApi, key, jwt);
+    if (res && status === 200) {
+        yield put(actions.item.leaderboard.success({key, leaderboard: res.testtuples}));
+    }
+    else {
+        console.error(error, status);
```

Comment:
you wanted to console.error?

Reply:
Yes, this is not a mistake, this is how API errors are handled throughout the frontend.